window_manager/src/XWindow.cpp

```diff
@@ -211,27 +257,24 @@ void XWindow::start_window_loop() {
     }
 
     window_loop();
+    BOOST_LOG_TRIVIAL(debug) << "Leaving " << __func__ ;
```

Comment:
If I'm not mistaken this is a compiler-specific macro. Shield it with a compiler detection macro.